### ISSUE TITLE
Semantic structure improvements

### ISSUE THREAD

**Issue by edsilv** [UniversalViewer/universalviewer#750]
ARIA-landmarks.

H1 is used for download menu (maybe elsewhere), should be h2.

**Comment by edsilv** [comment]
Make search result "teardrops" and thumbnails buttons so they appear in tab index?

**Comment by edsilv** [comment]
"Contents" and "More Information" should also be buttons.

**Comment by edsilv** [comment]
Make all `span` tags with `class="label"` a `label`

**Comment by crhallberg** [comment]
It's also a matter of if we want a flat hierarchy

```
Navigation
Index
Thumbnails
Image Viewer/PDF Viewer
Search within
Download
```

or a richer depth
```
UV/Content viewer
    Navigation
    Content
        Index
        Thumbnails
    Image Viewer/PDF Viewer
    Search within
    Download
```

**Comment by demiankatz** [comment]
Definitely interested to hear what @edsilv thinks about this, but I wonder if the flat hierarchy is best for starters because it's simpler and easier to apply consistently, and we can always refine it in future if more subtlety is needed.

**Comment by edsilv** [comment]
Agree that flat hierarchy sounds good if it's simpler

**Comment by edsilv** [comment]
Add title to embedding iframe: https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html

**Comment by edsilv** [comment]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

> In HTML5, all ARIA attributes validate. The new landmark elements (<main>, <header>, <nav> etc) have built-in ARIA roles, so there is no need to duplicate them.

I think it would be better to use semantic HTML5 instead of ARIA regions?

**Comment by demiankatz** [comment]
@edsilv, yes, if HTML5 can do the job, don't use ARIA -- it should only be a backup for edge cases where the native support can't fully express the semantics.

**Comment by LlGC-szw** [comment]
All issues will be triaged for further investigation or closure by the 28 September 2023. If your issue is still relevant and would like for it be investigated further please comment by 14 September 2023.
